addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -465,6 +464,29 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasDescriptionArea() {
+            return Boolean(
+                this.thread &&
+                (this.thread.description ||
+                this.thread.isChannelDescriptionChangeable)
+            );
+        }
+
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsDescriptionEditable() {
+            return Boolean(
+                this.messaging.currentUser &&
+                this.messaging.currentUser.isInternalUser
+            );
+        }
```

Comment:
looks like a (less complete) duplicate of `isChannelDescriptionChangeable`, we probably don't need this extra field, and just access the one from thread instead